docs/PyOp.md

```diff
@@ -0,0 +1,71 @@
+# Python Operator 
+To facilitate Python coders on model developing, onnxruntime provides a way to invoke operators implemented in Python.
+
+## Usage
+Step 1, build onnxruntime with“--config Release --enable_language_interop_ops --build_shared_lib”and override existing onnxruntime binary with the latest, then copy onnxruntime_pywrapper.dll or libonnxruntime_pywrapper.so or libonnxruntime_pywrapper.dylib to the path where onnxruntime binary is placed.
+Note that it is suggested to compile within the Python environment where inferencing will happen. For example, if inferencing will happen in a conda env named myconda1, please compile the binary within that environment as well.
```

Comment:
A simple figure to show how the pyop is interactive with onnxruntime will much help the end user to get the while idea.

Reply:
Good suggestion, let me add it.